ScoutSuite/providers/aws/facade/s3.py

```diff
@@ -204,6 +216,27 @@ def _set_s3_bucket_secure_transport(self, bucket: {}):
             print_exception('Failed to evaluate bucket policy for %s: %s' % (bucket['Name'], e))
             bucket['secure_transport'] = None
 
+    def get_s3_public_access_block(self, account_id):
+        # We need a region to generate the client
+        # However, the settings are global, so they are not region-dependent
+        region = 'eu-west-1'
```

Comment:
I haven't tested this out yet, so just dropping a quick not.
Hard-coding regions has bit us repeatedly in the past, so definitely an anti-pattern. 
At the very least, we should use `us-east-1`, as it can't be disabled - see https://github.com/nccgroup/ScoutSuite/pull/705

Reply:
See https://github.com/nccgroup/ScoutSuite/blob/develop/ScoutSuite/providers/aws/facade/s3.py#L14, which will take the region configured in the session if non is provided.